### DISCUSSION TITLE
Design Inspirations ✨

### DISCUSSION THREAD

**Discussion by egdiala** [NenLabs/nenlabs-website#2] [Ideas]
What's up guys @NenLabs/design 
So we'll be having NenUI website and Nenlabs organisation website

**Comment by egdiala** [comment]
> What's up guys @NenLabs/design So we'll be having NenUI website and Nenlabs organisation website

For the nenlabs organisation website, I want us to come up with a design similar but not exactly like that of [NuxtLabs](https://nuxtlabs.com/). If you find any other inspiration online, feel free to drop them here.

**Reply by onis77** [reply]
@CodeFreakk work dey

**Reply by egdiala** [reply]
It's actually going to be just one page 🙂

**Comment by egdiala** [comment]
I already bought domain name for nen labs and would like to get it up real soon, then update our projects there as time goes on 😊

**Reply by onis77** [reply]
nice, we'll start in earnest

**Reply by egdiala** [reply]
Aii

**Comment by CodeFreakk** [comment]
Omo..work choke for my end for now.

Sent from Yahoo Mail on Android 
 
  On Mon, Mar 13, 2023 at 2:27 PM, Abdulsalam ***@***.***> wrote:   


nice, we'll start in earnest

**Reply by egdiala** [reply]
No worries, you guys can start whenever you're free.
